packages/application-generic/src/usecases/trigger-event/trigger-event.usecase.ts

```diff
@@ -57,7 +58,8 @@ export class TriggerEvent {
     private processTenant: ProcessTenant,
     private logger: PinoLogger,
     private analyticsService: AnalyticsService,
-    private mapTriggerRecipients: MapTriggerRecipients
+    private mapTriggerRecipients: MapTriggerRecipients,
+    private subscriberProcessQueueService: SubscriberProcessQueueService
```

Comment:
Adding this lines 52-54 are not needed anymore. Those dependencies are moved to SubscriberJobBound use case, right?